contracts/AttestationRegistry.sol

```diff
@@ -0,0 +1,31 @@
+// SPDX-License-Identifier: GPL-3.0-only
+pragma solidity ^0.6.12;
+
+contract AttestationRegistry {
+    struct Attestation {
+        bytes32 id; // Holds a 256-bit hash
+        bytes32 publicKey; // Holds an ed25519 key
+        bytes1[64] signature; // Holds an ed25519 signature
```

Comment:
Alternativley

```suggestion
        byte[64] signature; // Holds an ed25519 signature
```

Reads slightly better in my opinion.

Reply:
Agreed but I think prettier did that. Happy to change the lint though if you know what it is..

Reply:
Confirmed, it's `prettier` :unamused:

Reply:
Ok. Then let’s keep it